pkg/config/containers.conf

```diff
@@ -627,6 +627,15 @@ default_sysctls = [
 #
 #user = "core"
 
+# Default host directories from the host to be mounted as volumes into the VM. 
```

Comment:
i think `Host directories to be mounted as volumes into the VM by default` might capture your intent?